roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -4,6 +4,25 @@
     - _cifmw_libvirt_manager_layout is not defined
   ansible.builtin.include_tasks: generate_layout.yml
 
+# TODO(hjensas): We should have a dedicated role for firewall
+- name: Enable forwarding in the libvirt zone
+  when: cifmw_libvirt_manager_firewalld_zone_libvirt_forward | default(false) | bool
+  become: true
+  notify: Restart firewalld
```

Comment:
so you're counting on the existing `flush_handlers` ? Are there running at the right time for you?

Reply:
right, it worked - but better be safe. I added a flush handler close to these tasks.
Good catch!